src/core/Link/Link/Link.md

```diff
@@ -1,122 +1,140 @@
-    underline="initial"
-  >
-    Not visited link
-  </Link>
-
-  <Link className="test-classname" href="#" underline="initial">
-    Visited link
-  </Link>
-
-  <Link href="https://www.notvisitedlink.com/" underline="hover">
-    Link without underline
-  </Link>
-
-  <Link variant="accent" href="https://www.notvisitedlink.com/">
-    Accented link
-  </Link>
-
-  <Link
-    variant="accent"
-    href="https://www.notvisitedlink.com/"
-    smallScreen
-  >
-    Small text accented link
-  </Link>
-</>;
+The `<Link>` component applies Suomi.fi styles to HTML `<a>` elements.
+
+Link should be used when you want navigate to a different URL, provide a way to bookmark a page or open it to a new tab.
+
+If you want to trigger an action that does not change the URL use the [Button](#/Components/Button) component instead.
+
+Examples:
+
+- [Basic use](/#/Components/Link?id=basic-use)
+- [Underline](/#/Components/Link?id=underline)
+- [Small screen](/#/Components/Link?id=small-screen)
+- [Accented](/#/Components/Link?id=accented)
+- [Skip link](/#/Components/Link?id=skip-link)
+- [External link](/#/Components/Link?id=external-link)
+- [Router link](/#/Components/Link?id=router-link)
+
+<div style="margin-bottom: 5px">
+  [Props & methods (Link)](/#/Components/Link?id=props--methods)
+</div>
+<div style="margin-bottom: 5px">
+  [Props & methods (SkipLink)](/#/Components/Link?id=skiplink)
+</div>
+<div style="margin-bottom: 5px">
+  [Props & methods (ExternalLink)](/#/Components/Link?id=externallink)
+</div>
+<div style="margin-bottom: 40px">
+  [Props & methods (RouterLink)](/#/Components/Link?id=routerlink)
+</div>
+
+### Basic use
+
+```jsx
+import { Link } from 'suomifi-ui-components';
+
+<Link href="#">Contact information</Link>;
+```
+
+### Underline
+
+Use the `underline` prop the control when the link gets underlined. Default value is `'hover'`.
+
+```jsx
+import { Link } from 'suomifi-ui-components';
+
+<Link href="#" underline="initial">
+  Contact information
+</Link>;
+```
+
+### Small screen
+
+Use small screen styles on narrower screen sizes.
+
+```jsx
+import { Link } from 'suomifi-ui-components';
+
+<Link href="#" smallScreen>
+  Contact information
+</Link>;
+```
+
+### Accented
+
+`variant="accent"` can be used to provide the Link more visual emphasis.
+
+```jsx
+import { Link } from 'suomifi-ui-components';
+
+<Link href="#" variant="accent">
+  Contact information
+</Link>;
 ```
 
 ### Skip link
 
-Show link for keyboard navigation.
+The `<SkipLink>` component is used to quickly jump to different parts of the page. It is inteded for keyboard users.
 
-- Should be the first element that can gain focus on the page with the tab.
-- The target container should have `tabIndex={-1}` to handle better the functionality with screenreaders.
+- Should be the first element that can gain focus on the page with Tab.
+- The target container should have `tabIndex={-1}` to better handle the functionality with screenreaders.
```

Comment:
Based on my testing and experience, this is not necessarily true, and can cause more issues than it fixes. If the target element is a heading, it works fine, but if e.g. the `main` element or something else with a lot of content has `tabindex="-1"`, it can cause some unexpected behaviour when using the page with a keyboard and mouse together.

Reply:
Adjusted the text somewhat. 

If the target is a simple element like a heading, consider applying tabIndex={-1} to it to better handle the functionality with screen readers.